Android Java quick start: "Could not find any version that matches io.grpc:grpc-core:[1.30.2]"

@ejona86 @voidzcy: in working through the [Android Java Quick Start > Run a gRPC application](https://grpc.io/docs/languages/android/quickstart/#run-a-grpc-application), I'm hitting the following issue on step 3:

```console
helloworld > ../../gradlew installDebug

> Configure project :app
The com.google.protobuf plugin was already applied to the project: :app and will not be applied again after plugin: android
Checking the license for package Android SDK Build-Tools 28.0.3 in /Users/chalin/Library/Android/sdk/licenses
License for package Android SDK Build-Tools 28.0.3 accepted.
Preparing "Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)".
"Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)" ready.
Installing Android SDK Build-Tools 28.0.3 in /Users/chalin/Library/Android/sdk/build-tools/28.0.3
"Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)" complete.
"Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)" finished.
Checking the license for package Android SDK Platform 27 in /Users/chalin/Library/Android/sdk/licenses
License for package Android SDK Platform 27 accepted.
Preparing "Install Android SDK Platform 27 (revision: 3)".
"Install Android SDK Platform 27 (revision: 3)" ready.
Installing Android SDK Platform 27 in /Users/chalin/Library/Android/sdk/platforms/android-27
"Install Android SDK Platform 27 (revision: 3)" complete.
"Install Android SDK Platform 27 (revision: 3)" finished.
WARNING: API 'variant.getJavaCompile()' is obsolete and has been replaced with 'variant.getJavaCompileProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variant.getJavaCompile(), use -Pandroid.debug.obsoleteApi=true on the command line to display a stack trace.

> Task :app:preDebugBuild FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:preDebugBuild'.
> Could not resolve all files for configuration ':app:debugCompileClasspath'.
   > Could not find any version that matches io.grpc:grpc-core:[1.30.2].
     Versions that do not match:
       - 1.30.1
       - 1.30.0
       - 1.29.0
       - 1.28.1
       - 1.28.0
       - + 67 more
     Required by:
         project :app > io.grpc:grpc-okhttp:1.30.2

...

BUILD FAILED in 33s
1 actionable task: 1 executed
```

Actually, before running that step I set `ANDROID_HOME`:

```console
$ export ANDROID_HOME=/Users/chalin/Library/Android/sdk
```

otherwise I was making even less progress.

Any idea why it isn't finding `io.grpc:grpc-core` v1.30.2?

--- a/config.yaml
+++ b/config.yaml
@@ -7,7 +7,7 @@ pygmentsCodeFences: true
 params:
   locale: en_US
   grpc_release_tag: v1.30.0
-  grpc_java_release_tag: v1.30.2
+  grpc_java_release_tag: v1.30.1
   grpc_go_release_tag: v1.30.0
   font_awesome_version: 5.12.1
   description: A high-performance, open source universal RPC framework

